File: src/main/java/aurocosh/divinefavor/common/util/SlotData.java
package aurocosh.divinefavor.common.util;

import net.minecraft.item.ItemStack;

public class SlotData {
    public final int slotIndex;
    public final ItemStack stack;

    public SlotData(int slotIndex, ItemStack stack) {
        this.slotIndex = slotIndex;
        this.stack = stack;
    }

    public boolean isValid(){
        return slotIndex > 0 || slotIndex < InventoryIndexes.Offhand.getValue();
    }
}

File: src/main/java/aurocosh/divinefavor/common/util/SlotData.kt
package aurocosh.divinefavor.common.util

import net.minecraft.item.ItemStack

data class SlotData(val slotIndex: Int, val stack: ItemStack) {
    val isValid: Boolean
        get() = slotIndex > 0 || slotIndex < InventoryIndexes.Offhand.value
}
